cool-core/src/main/java/com/nus/cool/core/cohort/refactor/birthselect/BirthSelectionContext.java

```diff
@@ -82,6 +82,13 @@ public boolean isUserSelected(String userId) {
     return userSelected.containsKey(userId);
   }
 
+  /**
+   * reset user's birthEvent.
+   */
+  public void resetUserSelected(String userId) {
```

Comment:
why call this function the reset function? It should be the remove function from the code meaning.